load.zsh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env zsh
+
+source /usr/share/zsh/plugins/zsh-autosuggestions/zsh-autosuggestions.zsh
```

Comment:
Leave some documentation how to install them.

Reply:
Done.